### PR TITLE
Fix spe char

### PR DESCRIPTION
**Pull request by yann300**
- rework on error/warning renderer
 - switch staticanalysis warning to be a dom node

### CODE REVIEW

File: src/app/files/fileManager.js [hunk 1]
```diff
@@ -98,7 +99,7 @@ class FileManager {
     $filesEl.find('.file').remove()
 
     for (var file in this.tabbedFiles) {
-      $filesEl.append($('<li class="file"><span class="name">' + file + '</span><span class="remove"><i class="fa fa-close"></i></span></li>'))
```
Comment (chriseth) [inline src/app/files/fileManager.js]: Oh wow, so I guess this is place of the bug, isn't it?

File: src/app/ui/renderer.js [hunk 2]
```diff
@@ -76,48 +76,72 @@ function Renderer (appAPI) {
...
+  if (!opt.noAnnotations && errLocation) {
+    this.appAPI.error(errLocation.errFile, {
+      row: errLocation.errLine,
+      column: errLocation.errCol,
+      text: text,
+      type: opt.type
     })
   }
 
+  var $pre = $(opt.useSpan ? '<span />' : '<pre />').html(message)
```
Comment (chriseth) [inline src/app/ui/renderer.js]: Is there no way to do that with yoyo?

File: src/app/files/fileManager.js (hunk 1 above)
Comment (yann300) [reply src/app/files/fileManager.js]: yes it is

File: src/app/ui/renderer.js (hunk 2 above)
Comment (yann300) [reply src/app/ui/renderer.js]: yes but didn't want to change everything, and this part is not going to be harmful
I can use yoyo here though

**Pull request merged**